solana-programs/staking-bridge/programs/staking-bridge/src/lib.rs

```diff
@@ -122,6 +141,40 @@ pub struct CreateStakingBridgeBalancePda<'info> {
     pub system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+#[instruction(staking_bridge_pda_bump: u8)]
+pub struct CreateStakingBridgeBalanceAtas<'info> {
+    #[account(
+        seeds = [b"staking_bridge".as_ref()],
+        bump = staking_bridge_pda_bump,
+    )]
+    /// CHECK: This is the PDA owned by this program. This account holds both SOL USDC and SOL AUDIO. It is used to swap between the two tokens. This PDA is also used to transfer SOL AUDIO to ETH AUDIO via the wormhole.
+    pub staking_bridge_pda: UncheckedAccount<'info>,
```

Comment:
I think some opportunity for naming consistency. we don't call this "pda" in payment router